solr/packaging/test/README.md

```diff
@@ -0,0 +1,57 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ -->
+
+# bin/solr Tests
+
+This directory contains tests for the `bin/solr` command-line scripts.
+
+All tests in this project use the [BATS](https://bats-core.readthedocs.io/en/stable/index.html) framework.
+
+## Running Tests
+
+Tests can be run via `./gradlew bats`.
```

Comment:
[-1] Should the task name here be `integrationTests`?  That's the task that works for me from the CLI, and the one that I see in `solr/packaging/build.gradle`